demos/controlgroup/default.html

```diff
@@ -0,0 +1,88 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>jQuery UI Controlgroup - Default Functionality</title>
+	<link rel="stylesheet" href="../../themes/base/all.css">
+	<script src="../../external/jquery/jquery.js"></script>
+	<script src="../../ui/core.js"></script>
+	<script src="../../ui/widget.js"></script>
+	<script src="../../ui/button.js"></script>
+	<script src="../../ui/checkboxradio.js"></script>
+	<script src="../../ui/menu.js"></script>
+	<script src="../../ui/position.js"></script>
+	<script src="../../ui/selectmenu.js"></script>
+	<script src="../../ui/controlgroup.js"></script>
+	<link rel="stylesheet" href="../demos.css">
+	<script>
+	$(function() {
+		$( ".controlgroup" ).controlgroup()
+		$( ".controlgroup-vertical" ).controlgroup({
+			"direction": "vertical"
+		});
+	});
+	</script>
+	<style>
+		.ui-controlgroup-vertical {
+			width: 300px;
+		}
+		.ui-controlgroup-vertical select {
+			width: 100%;
+		}
+		.ui-controlgroup.ui-controlgroup-vertical > button.ui-button {
+			text-align: center;
+		}
+	</style>
+</head>
+<body>
+<div class="widget">
+	<h1>Controlgroup</h1>
+	<fieldset>
+		<legend>Rental Car</legend>
```

Comment:
This doesn't make a lot of sense. These controls belong into one form, but not on one tightly grouped line. Only the transmission makes sense to have in a controlgroup, since that's an either-or choice.